<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerViewDialog</class>
 <widget class="QDialog" name="CustomerViewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>869</width>
    <height>630</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>869</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EasyManager-CustomerView</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
font:bold 15px;
color:white;
}


QDialog
{
background-image:url(&quot;:/Theme/BackImage/DialogBack_2.jpg&quot;)
}</string>
  </property>
  <widget class="QComboBox" name="CustomerBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>101</y>
     <width>711</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:bold 15px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>选择会员</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>联系方式：</string>
   </property>
  </widget>
  <widget class="QLabel" name="ContanctLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>581</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>581</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>注册时间：</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>581</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="RegiteTimeLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>571</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>520</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>当前余额：</string>
   </property>
  </widget>
  <widget class="QLabel" name="AccountLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>510</y>
     <width>581</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>530</y>
     <width>581</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
